From: Benjamin Drung <benjamin.drung@canonical.com>
Date: Wed, 26 Aug 2026 16:28:45 +0200
Subject: fix(chrony): support _chrony username

The test 61 chrony fails on Debian/Ubuntu and this failure can be seen
in the logs:

```
dracut-cmdline[278]: chown: invalid user: 'chrony:'
[...]
chronyd[461]: Wrong owner of /run/chrony (UID != 988)
```

The username used on Debian/Ubuntu is `_chrony`.

Partially fixes: https://github.com/dracut-ng/dracut/issues/2541
Origin: upstream, https://github.com/dracut-ng/dracut/commit/5ee731b62223d63269d8dc3be3cdc798869264d1
---
 modules.d/74chrony/parse-ntp.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/74chrony/parse-ntp.sh b/modules.d/74chrony/parse-ntp.sh
index 0446af6..150b0a9 100755
--- a/modules.d/74chrony/parse-ntp.sh
+++ b/modules.d/74chrony/parse-ntp.sh
@@ -45,7 +45,7 @@ parse_ntp_source() {
 }
 
 mkdir -p -m 0750 /run/chrony
-chown chrony: /run/chrony
+chown "$(get_user _chrony chrony):" /run/chrony
 mkdir /run/chrony/dracut.sources.d
 
 for _i in $(getargs rd.ntp); do
